JustTrack/EventsGenerator/main.swift

```diff
@@ -419,6 +434,45 @@ func generateObjectStructs(_ objects: [Any]) throws -> String {
     }
 }
 
+func determineItemType(item: String) -> String {
```

Comment:
This function name is misleading - we are not determining the type here, just stripping suffixes.

Reply:
Done